Cargo.toml

```diff
@@ -8,10 +8,10 @@ members = [
     ## Middleware
     "middleware/template",
     "middleware/under_development/diesel",
-    "middleware/jwt",
```

Comment:
I know we had to temporarily comment this out on #308 - was there a reason to remove it here as well? I liked leaving it as commented as reminder to bring it back when dependencies can be sorted out.